notebook/tests/selenium/test_deletecell.py

```diff
@@ -0,0 +1,74 @@
+import os
+import pytest
+from .utils import Notebook
+
+@pytest.fixture
+def notebook(authenticated_browser):
+    return Notebook.new_notebook(authenticated_browser)
+
+def get_cells_contents(nb):
+    JS = 'return Jupyter.notebook.get_cells().map(function(c) {return c.get_text();})'
+    return nb.browser.execute_script(JS)
+
+def set_cell_metadata(nb, index, key, value):
+    JS = 'Jupyter.notebook.get_cell({}).metadata.{} = {}'.format(index, key, value)
+    return nb.browser.execute_script(JS)
+
+def cell_is_deletable(nb, index):
+    JS = 'return Jupyter.notebook.get_cell({}).is_deletable();'.format(index)
+    return nb.browser.execute_script(JS)
+
+def delete_cell(notebook, index):
+    notebook.focus_cell(index)
+    notebook.to_command_mode
+    notebook.current_cell.send_keys('dd')
+
+def test_delete_cells(notebook):
+    print('testing deleteable cells')
```

Comment:
We shouldn't need this print() - pytest should tell us which test is which when it runs them.